server/sample.env

```diff
@@ -1,3 +1,3 @@
 JWT_SECRET=
 MONGO_URI=
-TEAM_NAMES=Bonnie,Sina
\ No newline at end of file
+TEAM_NAMES=Amita
```

Comment:
Similar comment as the last PR about rebasing/adjust the base of the PR :)

Reply:
Done the changes